react/__mocks__/react-intl.js

```diff
@@ -0,0 +1,13 @@
+import React, { Fragment } from 'react'
```

Comment:
Probably ou can avoid mocking `react-intl` using something like this https://github.com/vtex-apps/shelf/blob/master/react/__test__/Shelf.test.js#L136